src/tests/ftest/util/daos_utils.py

```diff
@@ -89,6 +89,10 @@ def container_create(self, pool, sys_name=None, cont=None,
             CommandFailure: if the daos container create command fails.
 
         """
+        if properties is None:
+            properties.append('rf_lvl:1')
+        else:
+            properties.append(',rf_lvl:1')
```

Comment:
Your comment makes sense. Since no tests currently set `rf_lvl`, I think conceptually this is okay for now. But this will break some things, since `properties` is a string and therefor doesn't have an `append` method. This should fix and make it more future proof:
```suggestion
        # Default to previous rf_lvl behavior when not specified
        if properties:
            if 'rf_lvl' not in properties:
                properties += ',rf_lvl:1'
        else:
            properties = 'rf_lvl:1'
```

Reply:
ah, ok, good, I'll fix it, thx